src/nodes.coffee

```diff
@@ -1296,6 +1314,115 @@ exports.LineComment = class LineComment extends Base
 
 #### Call
 
+exports.CSXElement = class CSXElement extends Base
+  constructor: ({@tagName, @attributes, @content}) ->
+    super()
+
+  children: ['tagName', 'attributes', 'content']
```

Comment:
I'm not clear enough on how exactly `children` gets used to know if this is necessary? But seems like the right value for `children` here

Reply:
`children` is used by the node traversal methods, like `unwrap` and `traverseChildren` and so on. So basically any nodes that this element contains would be children.